js/classes/player.js

```diff
@@ -12,13 +13,21 @@ const weaponSlots = {
   lHand: null,
   rHand: null
 }
+
 class Player extends Character {
   constructor(name, stats) {
     super(name, stats);
     this._armor = armorSlots;
     this._weapon = weaponSlots;
+
+    this._icon = 'user';
+
+    //build an icon
+    this._createIcon = buildElement('touch-icon', {class: 'player'}, this.getInfo);
```

Comment:
the character class actually has an "elements" object now. In the constructor just write

`this._elements['createIcon'] = buildElement(...)`

Reply:
done